panel/reactive.py

```diff
@@ -956,7 +966,7 @@ def __init__(self, **params):
         super().__init__(**params)
         self._old = None
     
-    def _update_selection(self, indices):
+    def _update_selection(self, indices: List[int]):
```

Comment:
I believe this is correct. But its hard to infer looking at the code. I.e. the typehint will actually help the next developer  :-)

Maybe its even allowed to be `Optional`, who knows?

Reply:
I think that's right.